### PR TITLE
Upgrade kubevela io

### PR DESCRIPTION
**Pull request by oeular**
### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` or `[Trait]`).
- [x] Updated/Added any relevant [documentation] and [examples].
- [ ] Unit/E2E Tests added.

### CODE REVIEW

File: addons/kubevela-io/resources/deployment/kubevela-io-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
```
Comment (barnettZQG) [inline addons/kubevela-io/resources/deployment/kubevela-io-deployment.yaml:1]: Why not use `webservice` component?

Comment (oeular) [reply addons/kubevela-io/resources/deployment/kubevela-io-deployment.yaml:1]: There are several reasons.
- Users may need the local docs  first to install Vela.
- When there is problem with the Vela, the doc may not work well.(That's not a good news.)

Comment (oeular) [reply addons/kubevela-io/resources/deployment/kubevela-io-deployment.yaml:1]: - If users need the doc first, they can use "kubectl apply" to install Vela  with the resources.(And then they can install Vela according the doc.)
- If Vela is something wrong, user can use "kubectl apply" to reinstall Vela with the resources.

Comment (barnettZQG) [reply addons/kubevela-io/resources/deployment/kubevela-io-deployment.yaml:1]: Then there is a concern that whether Vela will affect already deployed applications.

Comment (wonderflow) [reply addons/kubevela-io/resources/deployment/kubevela-io-deployment.yaml:1]: It's fine for the initial version, and this is also a good example about pure K8s object to be as addons.

**Review by wonderflow — APPROVED** [review]

File: addons/kubevela-io/resources/deployment/kubevela-io-deployment.yaml (hunk 1 above)
Comment (oeular) [reply addons/kubevela-io/resources/deployment/kubevela-io-deployment.yaml:1]: @barnett The problem for the resource versions will be a big concern in Vela. Here is an issue, https://github.com/oam-dev/kubevela/issues/3624#issue-1199716391

**Pull request merged**